src/monster.h

```diff
@@ -87,7 +87,7 @@ class monster : public Creature
         bool is_monster() const override {
             return true;
         }
-
+        std::string facing = "";
```

Comment:
There are two values, possibly a third "none/undefined", for facing, no reason to use a string here. Use an enumeration such as:
```
enum FacingDirection{
    FD_NONE = 0,
    FD_LEFT = 1,
    FD_RIGHT = 2
};
```
Then replace `std::string facing = "";` with `FacingDirection facing = FD_NONE;`

Furthermore, `facing` could be defined in `class Creature` (creature.h) which is a common parent between both `class Monster` and `class Player`, with `enum FacingDirection` defined somewhere publicly visible for use in comparisons.